ironic/templates/compute/daemonset.yaml.j2

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+spec:
+  template:
+    metadata:
+      labels:
+        app: compute-node
+    spec:
+      nodeSelector:
+        # TODO:
+        # {{ deployment.compute_node_label }}: enabled
+        mateuszb: enabled
+      securityContext:
+        runAsUser: 0
+      hostNetwork: true
+      hostPID: true
+      dnsPolicy: ClusterFirst
+      containers:
+        - name: nova-compute-ironic
+          image: {{ deployment.image.compute }}
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          env:
+            - name: INTERFACE_NAME
+              value: {{ network.minion_interface_name }}
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: COMMAND
+              value: "nova-compute"
+            - name: DEPENDENCY_JOBS
+              value: "ironic-init,ironic-post"
+            - name: DEPENDENCY_SERVICE
+              value: "keystone-api,glance-api,nova-api"
+            - name: DEPENDENCY_CONFIG
+              value: "/etc/nova/nova.conf,/etc/resolv.conf"
```

Comment:
can we mount /etc/resolv.conf as a configmap, as done here: https://github.com/stackanetes/stackanetes/pull/99